services/apps/src/apps/clients/iplant_groups.clj

```diff
@@ -18,6 +18,30 @@
 (def grouper-analysis-permission-def-fmt "iplant:de:%s:analyses:analysis-permission-def")
 (def grouper-analysis-resource-name-fmt "iplant:de:%s:analyses:%s")
 
+(def ^:private permission-precedence
+  (into {} (map-indexed (fn [i v] (vector v i)) ["own" "write" "read"])))
```

Comment:
Yeah, I guess it might be a little more readable that way. I liked the compact representation when I wrote this line of code, but making it explicit sounds fine to me too.